src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,30 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post: { title, body, user, comments } }) => (
```

Comment:
The prop types are not defined. It's better to define the PropTypes for the PostInfo component to maintain the type safety.